src/resources/postcss/views/_loader.pcss

```diff
@@ -0,0 +1,18 @@
+.tribe-events {
+	.tribe-events-view-loader {
+		align-items: center;
+		justify-content: center;
+		display: flex;
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		z-index: var(--layer-spinner-container);
+		background: var(--color-background-transparent);
+
+		&__spinner {
```

Comment:
this works, but I'm gonna be picky and say let's move this out to it's own fully written out class `.tribe-events-view-loader__spinner`. It just makes it easier to follow and `ctrl+f`.